### PR TITLE
GEN-1630 / Adjust spacing on cart page

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Spacing was off on Cart page

Comment on how we use `Space`. For me it get's a bit hard to follow the spacing on a page when using multiple levels of nested `Space` components 🙃 

## Justify why they are needed
Design review feedback

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3765** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3765?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #3766** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3766?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3767** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3767?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3765?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]

**Comment by gustaveen** [comment]
### Merge activity

* **Dec 20, 4:59 AM**: @@gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3765).
